man/pam_systemd_home.xml

```diff
@@ -146,7 +146,7 @@ password  required   pam_deny.so
 -session  optional   pam_loginuid.so
 <command>-session  optional   pam_systemd_home.so</command>
 -session  optional   pam_systemd.so
-session   required   pam_unix.so</programlisting>
+session   required   pam_unix.so no_pass_expiry</programlisting>
```

Comment:
looking at the sources, this sounds unnecessary, as the flag is not checked in the "session" hook, but only in the account hook